rust/src/test.rs

```diff
@@ -8,6 +8,7 @@ pub use crate::flags::*;
 /// Function which indicates if the test should be skipped by returning an error.
 pub type Guard = fn(&Config, &Path) -> Result<(), anyhow::Error>;
 
+#[derive(Clone, Copy)]
```

Comment:
```suggestion
```

Reply:
We can't remove this.  We actually do need Copy for TestFn, because of the copy in main.rs at line 119.